sdk/python/orm/sqlalchemy.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: SQLAlchemy
+sidebarTitle: SQLAlchemy
+description: Configure SQLAlchemy to work with your Turso database
+---
+
+![SQLAlchemy Quickstart](/images/guides/sqlalchemy-banner.png)
+
+## Prerequisites
+
+Before you start, make sure you:
+
+- [Install the Turso CLI](/cli/installation)
+- [Sign up or login to Turso](/cli/authentication#signup)
+
+<Steps>
+
+<Step title="Install the libSQL dialect for SQLAlchemy">
+
+```bash
+pip install sqlalchemy-libsql
+```
+
+</Step>
+
+<Step title="Configure database credentials">
+
+Get the database URL:
+
+```bash
+turso db show --url <database-name>
+```
+
+Get the database authentication token:
+
+```bash
+turso db tokens create <database-name>
+```
+
+Assign credentials to the environment variables inside `.env`.
+
+```bash
+TURSO_DB_URL="..."
+TURSO_DB_AUTH_TOKEN="..."
+```
+
+</Step>
+
+<Step title="Create database models">
+
+```python models.py
+from sqlalchemy import String
+from sqlalchemy.orm import DeclarativeBase
+from sqlalchemy.orm import Mapped
+from sqlalchemy.orm import mapped_column
+
+class Base(DeclarativeBase):
+    pass
+
+class Foo(Base):
+    __tablename__ = "foo"
+    id: Mapped[str] = mapped_column(primary_key=True)
+    bar: Mapped[str] = mapped_column(String(100))
+    def __repr__(self) -> str:
+        return f"Item(id={self.id!r}, bar={self.bar!r})"
+
+```
+</Step>
+
+<Step title="Construct SQLAlchemy URL for Turso">
+
+```py app.py
+from dotenv import load_dotenv
+from sqlalchemy import create_engine
+
+# Get environment variables
```

Comment:
We could probably remove these comments since it seems pretty clear what they do. Wdyt?